README.md

```diff
@@ -112,7 +104,7 @@ $ bundle exec rspec
 
 ## Authors
```

Comment:
[Optional] I suggest you use Author instead of Authors as this is a solo project

Reply:
Done.